assets/css/dark.css

```diff
@@ -12,7 +12,7 @@
     }
 
     .hero p {
-        color: #c3c3c3;
+        color: #848484;
```

Comment:
I feel like `#848484` is still too low (although, much better). Perhaps we could try something lighter?

Reply:
> I feel like `#848484` is still too low (although, much better). Perhaps we could try something lighter?

Yep, I agree, We should go with something above `AA` for contrast, I recommend `#A3A3A3`, it's almost `AAA` so, the contrast should be good for most users.

**Reference:** [Use Contrast](https://usecontrast.com/guide)
**Contrast Ratio for `#A3A3A3`:** 6.2:1
**How the text looks:** 
<img src="https://github.com/Vanilla-OS/website/assets/110247388/0d69f776-475c-4401-b530-f77e00e49860" width="160px">

```suggestion
        color: #A3A3A3;
```

Reply:
Even though it looks visually darker, `#A3A3A3` has better contrast than `#848484`
LGTM

`#A3A3A3`
<img width="1670" alt="Screenshot 2023-06-10 at 15 40 47" src="https://github.com/Vanilla-OS/website/assets/49836430/1821ec4b-6e75-4973-a0ef-c840163c3db4">

`#848484`
<img width="1655" alt="Screenshot 2023-06-10 at 15 41 19" src="https://github.com/Vanilla-OS/website/assets/49836430/d751cb86-2ca1-4390-ac46-bdf5ed28b929">